test/plugins/surround.test.ts

```diff
@@ -59,6 +77,7 @@ suite('surround plugin', () => {
     end: ['first |( line ).test'],
   });
 
+  /*
```

Comment:
Maybe we can stub `vscode.window.showInputBox` to make these tests possible?

Reply:
I have adapted the existing tests with a stub.